README.md

```diff
@@ -52,10 +52,40 @@ npm install @epic-web/cachified
 
 ```ts
 import { LRUCache } from 'lru-cache';
-import { cachified, CacheEntry } from '@epic-web/cachified';
+import { cachified, CacheEntry, Cache, totalTtl } from '@epic-web/cachified';
 
 /* lru cache is not part of this package but a simple non-persistent cache */
-const lru = new LRUCache<string, CacheEntry>({ max: 1000 });
+const lruInstance = new LRUCache<string, CacheEntry>({ max: 1000 });
+
+/* defines options for the set method for lru cache */
+interface LRUishCache extends Omit<Cache, 'set'> {
+  set(
+    key: string,
+    value: CacheEntry<unknown>,
+    options?: { ttl?: number; start?: number },
+  ): void;
+}
+
+/* creates a wrapper for the lru cache so that it can easily work with cachified and ensures the lru cache cleans up outdated values itself*/
+function lruCacheAdapter(lruCache: LRUishCache): Cache {
+  return {
+    set(key, value) {
+      const ttl = totalTtl(value?.metadata);
+      return lruCache.set(key, value, {
+        ttl: ttl === Infinity ? undefined : ttl,
+        start: value?.metadata?.createdTime,
+      });
+    },
+    get(key) {
+      return lruCache.get(key);
+    },
+    delete(key) {
+      return lruCache.delete(key);
+    },
+  };
+}
+
+const lru = lruCacheAdapter(lruInstance);
```

Comment:
Remove this stuff and just set `lru` to the cache instance itself. So something like:

```
const lru: Cache = {
  set() {},
  get() {},
  delete() {},
}
```

Of course you'll want to fill that in with working code and make sure the types are happy too. I think you'll want to use the Cache generic maybe.